.devcontainer/build.sh

```diff
@@ -0,0 +1,84 @@
+#!/bin/bash
+
+#
+#    Copyright (c) 2023 Project CHIP Authors
+#
+#    Licensed under the Apache License, Version 2.0 (the "License");
+#    you may not use this file except in compliance with the License.
+#    You may obtain a copy of the License at
+#
+#        http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS,
+#    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#    See the License for the specific language governing permissions and
+#    limitations under the License.
+#
+
+HERE="$(dirname "$0")"
+BUILD_VERSION="0.6.35"
+IMAGE_TAG="matter-dev-environment:local"
+USER_UID=$UID
+
+function show_usage {
+    cat <<EOF
+Usage: $0
+
+Build vscode dev environment docker image.
+
+Options:
+    -h,--help        Show this help
+    -t,--tag         Image tag
+    -u,--uid         User UIDa
+    --set-version    Build version
```

Comment:
What do you think about? 
```suggestion
    -v,--version    Build version
```
To be consistent with the rest of the options.

Reply:
In pretty much all scripts I see -v is to get the version. Maybe this being a build script make it different, but for me -v or --version makes me believe it is to get the version instead of setting it. If you feel it's fine then I'll do the change

Reply:
It's OK to get the version parameter. The help command shows the details about options.

Reply:
Done